manager/integration/tests/backupstore.py

```diff
@@ -0,0 +1,410 @@
+
+    if is_backupTarget_s3(backupstore):
+        return minio_delete_backup_cfg_file(client,
+                                            core_api,
+                                            volume_name,
+                                            backup_name)
+
+    elif is_backupTarget_nfs(backupstore):
+        return nfs_delete_backup_cfg_file()
+
+
+# TODO: implement nfs_delete_backup_cfg_file
+def nfs_delete_backup_cfg_file():
+    raise NotImplementedError
+
+
+def minio_delete_backup_cfg_file(client, core_api, volume_name, backup_name):
+    secret_name = backupstore_get_secret(client)
+    assert secret_name != ''
+
+    minio_api = minio_get_api_client(client, core_api, secret_name)
+    bucket_name = minio_get_backupstore_bucket_name(client)
+    minio_backup_cfg_file_path = minio_get_backup_cfg_file_path(volume_name,
+                                                                backup_name)
+
+    try:
+        minio_api.remove_object(bucket_name, minio_backup_cfg_file_path)
+    except ResponseError as err:
+        print(err)
+
+
+def backupstore_delete_volume_cfg_file(client, core_api, volume_name):  # NOQA
+    backupstore = backupstore_get_backup_target(client)
+
+    if is_backupTarget_s3(backupstore):
+        return minio_delete_volume_cfg_file(client,
+                                            core_api,
+                                            volume_name)
+
+    elif is_backupTarget_nfs(backupstore):
+        return nfs_delete_volume_cfg_file()
+
+
+# TODO: implement nfs_delete_volume_cfg_file
+def nfs_delete_volume_cfg_file():
+    raise NotImplementedError
+
+
+def minio_delete_volume_cfg_file(client, core_api, volume_name):
+    secret_name = backupstore_get_secret(client)
+    assert secret_name != ''
+
+    minio_api = minio_get_api_client(client, core_api, secret_name)
+    bucket_name = minio_get_backupstore_bucket_name(client)
+    minio_volume_cfg_file_path = minio_get_volume_cfg_file_path(volume_name)
+
+    try:
+        minio_api.remove_object(bucket_name, minio_volume_cfg_file_path)
+    except ResponseError as err:
+        print(err)
+
+
+def backupstore_create_dummy_in_progress_backup(client, core_api, volume_name):
+    backupstore = backupstore_get_backup_target(client)
+
+    if is_backupTarget_s3(backupstore):
+        return minio_create_dummy_in_progress_backup(client,
+                                                     core_api,
+                                                     volume_name,
+                                                     "backup-dummy")
+
+    elif is_backupTarget_nfs(backupstore):
+        return nfs_create_dummy_in_progress_backup()
+
+
+# TODO: implement nfs_create_dummy_in_progress_backup
+def nfs_create_dummy_in_progress_backup():
+    raise NotImplementedError
+
+
+def minio_create_dummy_in_progress_backup(client, core_api, volume_name, backup_name): # NOQA
+    dummy_backup_cfg_data = {"Name": "dummy_backup",
```

Comment:
create the dummy_backup_cfg file inside of backupstore_create_dummy_in_progress_backup
since it's identically for minio/nfs, then remove this step in between and call the backupstore_write_backup_cfg_file method directly from the dummy method.

Reply:
fixed